scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -111,6 +116,34 @@ class FossId(
 
             return projectName
         }
+
+        /**
+         * This function fetches credentials for [repoUrl] and insert them between the url scheme and the host. If no
+         * matching host is found the in netrc file, the [repoUrl] is returned untouched.
+         */
+        fun getCredentialsFromNetrc(repoUrl: String): String {
```

Comment:
Is this function only used internally? Then it could be *private*.

Reply:
I wanted to access it in testing but since I do not come further with testing I will make it `private`.